src/audio/volume.c

```diff
@@ -191,13 +191,14 @@ static struct comp_dev *volume_new(struct sof_ipc_comp *comp)
 
 	comp_set_drvdata(dev, cd);
 	work_init(&cd->volwork, vol_work, dev, WORK_ASYNC);
-
```

Comment:
I like to see a newline between blocks as it helps code readability.

Reply:
Your SoB line is incorrect, should be 
```text
Marcin Rajwa <marcin.rajwa@linux.intel.com>
```